docs/src/domains/steps/tls.md

```diff
@@ -1,52 +1,38 @@
 ---
-title: "(Optional) Configure a third-party TLS certificate"
+title: "Optional: Configure a third-party TLS certificate"
 weight: 3
 sidebarTitle: "Custom TLS"
 ---
 
 Platform.sh automatically provides all production environments with standard TLS certificates issued by [Let's Encrypt](https://letsencrypt.org/).
 No further action is required to use TLS-encrypted connections beyond [specifying HTTPS routes](../../define-routes/https.md).
 
-Alternatively, you may provide your own third-party TLS certificate from the TLS issuer of your choice.
+Alternatively, you can provide your own third-party TLS certificate from the TLS issuer of your choice.
 Consult your TLS issuer for instructions on how to generate an TLS certificate.
 
 A custom certificate isn't necessary for development environments.
 Platform.sh automatically provides wildcard certificates that cover all `*.platform.sh` domains, including development environments.
 
-{{< note >}}
+All kinds of certificates are supported including:
 
-The private key should be in the old style, which means it should start with `BEGIN RSA PRIVATE KEY`.
-If it starts with `BEGIN PRIVATE KEY`, it's bundled with the identifier for key type.
+* domain-validated certificates,
+* extended validation (EV) certificates,
+* high-assurance (HA) certificates,
+* wildcard certificates.
 
-To convert it to the old-style RSA key:
-
-```bash
-openTLS rsa -in private.key -out private.rsa.key
-```
-
-{{< /note >}}
+The use of HA or EV certificates is the main reason why you may wish to use a third-party issuer rather than the default certificate.
+If you use wildcard routes, you need a custom certificate.
```

Comment:
:hammer_and_wrench: I feel like the order of causation is somewhat confused here in that you can't use a wildcard route until you get a custom certificate, so it's not about you already using it, if that makes sense. Perhaps this?

```suggestion
Another reason you might need a custom certificate is for wildcard routes.
```

Reply:
Or this, given that this is where I'd expect the detailed information?
```suggestion
Let's Encrypt certificates also don't support wildcard routes,
so you need a custom certificate if you want to use one.
```